SwiftWisdom/Core/UIKit/UIViewController+Nibs.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIViewController+Nibs.swift
+//  SwiftWisdom
+//
+//  Created by Logan Wright on 1/19/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+    public static func ip_fromNib() -> Self {
+        let controller = self.init(nibName: ip_nibName, bundle: nil)
+        return controller
+    }
+}
+
+extension UIViewController {
```

Comment:
Should be, moving now.  They came from different times in a project and I hadn't combined them until now.